alerts/charts/templates/am-notification-template-configmap.yaml

```diff
@@ -5,7 +5,7 @@ kind: ConfigMap
 metadata:
   name: greenhouse-notification-templates
   labels:
-    plugin: alerts
+    pluginDefinition: alerts
     pluginconfig: {{ $.Release.Name }}
```

Comment:
```suggestion
    plugin: {{ $.Release.Name }}
```

Reply:
Ah ok, we might need to inform plugin owner, if that label changes...

Reply:
good point I will revert this for now. As this is not necessarily connected to the CRD itself but an internal of the chart